dfttk/workflows.py

```diff
@@ -679,6 +680,14 @@ def process_phonon_dos_YPHON(path: str):
         path (str): path to the folder containing all the phonon calculation folders. E.g. phonon_1, phonon_2, etc.
     """
 
+    # Configure logging
+    log_file_path = os.path.join(path, 'phonons_parallel.log')
```

Comment:
Maybe change to log_file_path = os.path.join(path, 'process_phonon_dos_YPHON.log') to reflect the function.

Reply:
sure.

Reply:
The issue is resolved for me if I use this in the beginning of the function:

# Reset logging configuration
    for handler in logging.root.handlers[:]:
        logging.root.removeHandler(handler)
        
    # Configure logging
    log_file_path = os.path.join(path, 'phonons_parallel.log')
    
    # If log_file_path already exists, delete it
    if os.path.exists(log_file_path):
        os.remove(log_file_path)
        
    logging.basicConfig(
        filename=log_file_path,
        level=logging.ERROR,  # Set the log level to ERROR
        format='%(asctime)s - %(levelname)s - %(message)s'
    )

Reply:
# And delete the empty file at the end

# Delete log_file_path if it is empty
        if os.path.exists(log_file_path) and os.path.getsize(log_file_path) == 0:
            os.remove(log_file_path)